docker/Dockerfile.syslog

```diff
@@ -5,7 +5,7 @@ FROM python:2.7-slim as scalyr-dependencies
 MAINTAINER Scalyr Inc <support@scalyr.com>
 
 RUN apt-get update && apt-get install -y build-essential
-RUN pip --no-cache-dir install --root /tmp/dependencies ujson yappi docker-py==1.10.3
+RUN pip --no-cache-dir install --root /tmp/dependencies ujson yappi requests==2.15.1 docker==4.1.0
```

Comment:
Eventually, to remove confusion, I think we should have multiple requirements.txt files:

- ``requirements.txt`` - file which contains dependencies the agent depends on which are not bundled with the agent directory. Think ujson, yappi, docker, etc. To be good citizen, we should also declare those in ``setup.py`` under requirements section. Granted, things get a bit more complicated with "optional" dependencies such as docker one which is only required when docker monitor is running...
- ``dev/test-requirements.txt`` - file which only contains dev / testing dependencies (right now we are mixing those and production one in the same file)

Those files should then also be used by those Dockerfiles.

Reply:
Maybe I  should create separate task/PR for this?